src/Sylius/Behat/Context/Setup/ShippingContext.php

```diff
@@ -69,18 +71,30 @@ public function getShippingMethodByName($shippingMethodName)
 
     /**
      * @Given store ships everything for free
+     * @Given /^store ships everything for free within ("([^"]*)" zone)$/
```

Comment:
Anyway, for those simple steps I prefer not using regexps:
`@Given store ships everything for free within :zone zone`
And then zone transformer with `@Transform :zone`